examples/image_annotate/definitions/annotated_imagesets.sqlx

```diff
@@ -0,0 +1,25 @@
+config {
+    type: "incremental"
+}
+
+SELECT
+  *
+FROM
+  ${ref("object_table")}
+WHERE
+  ml_annotate_image_status NOT LIKE "${constants.RETRYABLE_ERROR}"
+  ${
+      when(incremental(), `AND
+      ${constants.MERGE_COLUMN} NOT IN (
+      SELECT
+        ${constants.MERGE_COLUMN}
+      FROM
+        ${self()})
+      OR updated > (
```

Comment:
Where is field coming from? I don't see it listed in the output of ML.ANNOTATE_IMAGE

https://cloud.google.com/bigquery/docs/reference/standard-sql/bigqueryml-syntax-annotate-image

Reply:
Also, why do you need that bit given the above?